### ISSUE TITLE
References to New WCAG 2.2 definitions are setting "New" as title instead of the definition

### ISSUE THREAD

**Issue by rscano** [w3c/wcag#3511]
Perimeter definition has title="[New]">minimum bounding box</a>. title should be: "the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie."

**Comment by mbgower** [comment]
@rscano I assume you are referring to https://www.w3.org/TR/WCAG22/#dfn-perimeter

Are you advocating that the first line of the minimum bounding box be swapped in place of a link to the minimum bounding box definition?

**Comment by rscano** [comment]
In the code the title attribute is set as "[new]". For all other definition, when linked inside document, title attribute is an abstract of first sentence in glossary. So text should be:
<code>
<dd class="new">
	<p class="change">[New]</p>	
   <p>continuous line forming the boundary of a shape not including shared pixels, or the <a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="the smallest enclosing rectangle aligned to the horizontal axis within which all the points of a shape lie">minimum bounding box</a>, whichever is shortest.</p>
  <aside class="example" id="example-16"><div class="marker">Example</div><p>The perimeter calculation for a 2 CSS pixel perimeter around a rectangle is 4<em>h</em>+4<em>w</em>, where <em>h</em> is the height and <em>w</em> is the width. For a 2 CSS pixel perimeter around a circle it is 4𝜋<em>r</em>.</p></aside>
</dd>
</code>

**Comment by mbgower** [comment]
@iadawn is this "New" title being formed in code?

**Comment by iadawn** [comment]
This is added as part of the build process via the [wcag.js script](https://github.com/w3c/wcag/blob/869d568d59430d0237bef389cf34bb8db1eae1db/script/wcag.js#L77-L84).

This code will add in a `title` with the content from the first child element of the linked definition. If the linked definition happens to be new then the first child element is `<p class="change">New</p>`

**Comment by mbgower** [comment]
@iadawn same issue is flagged for different new definitions in all of the following:
#3499 
#3495 
#3496 
#3501 

These definitions already have a class of "new"; is it possible to use CSS to put the "NEW" text where you want positionally, while leaving it underneath the definition in the DOM (and thus not picked up by the script)?



![Image](https://github.com/w3c/wcag/assets/14298245/1644771d-0954-425e-9308-d5b22d80f990)

**Comment by rscano** [comment]
Sorry but my note wasn't for the NEW in paragraph but for the "new" as title inside the code:

`<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-2" title="**[New]**">minimum bounding box</a>`

**Comment by mbgower** [comment]
> Sorry but my note wasn't for the NEW in paragraph but for the "new" as title inside the code for title attribute:

Yes, but since that paragraph is the first child of the definition, its contents ("NEW") is being populated as the title value, as pointed out in [prior discussion](https://github.com/w3c/wcag/issues/3511#issuecomment-1845205918):
> This code will add in a title with the content from the first child element of the linked definition. If the linked definition happens to be new then the first child element is <p class="change">New</p>

**Comment by mbgower** [comment]
@kfranqueiro Is there anything we can do to address this issue that when one hovers over a definition that is identified as "new", the tooltip is showing up as "New" instead of as the definition.

<img width="981" height="236" alt="Image" src="https://github.com/user-attachments/assets/a6d0f5d2-fd8b-4a5b-a04f-e718471b9f97" />

@rscano In addition to this issue, you have opened the following issues:
https://github.com/w3c/wcag/issues/3499
https://github.com/w3c/wcag/issues/3495
https://github.com/w3c/wcag/issues/3496
https://github.com/w3c/wcag/issues/3501

Since all these issues are all a result of the same "New" string, and the resolution of one will lead to the resolution of all, I going to consolidate them into this one issue and close the others as duplicates. 

I have renamed this issue to reflect that.

**Comment by patrickhlauke** [comment]
@kfranqueiro @iadawn as the title tooltip won't work for keyboard users (though one could argue this is only a "nice to have" feature, rather than essential), would it perhaps make sense to just remove this "feature" altogether?

**Comment by bruce-usab** [comment]
I removed from the board, since we have the PR.
